x-pack/filebeat/module/cisco/ios/main_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package ios
+
+import (
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/elastic/beats/v7/testing/testflag"
+)
+
+func TestMain(m *testing.M) {
+	testflag.MustSetStrictPermsFalse()
```

Comment:
I'm not a huge fan of this approach, you're changing a global flag/state to all tests jut to "fix" something that can be fixed directly on the tests. Also as far as I know wasn't is not failing on CI.

Reply:
Using this flag/environment variable as suggest by Craig on the PR I did to make this test do not rely on the `umask` of the host they're running on. I added it here because: 1) I needed to make sure they pass and I was running them locally, 2) it's better to have them fixed for good

Reply:
I'm a bit confused here. You merged https://github.com/elastic/beats/pull/42914/, that fixes the permissions on both Go and Python integration tests. After that were there tests still failing. It looks redundant to me.

Reply:
I can only say when I run the test to make and test https://github.com/elastic/beats/pull/42914 all tests passed. Now tests in this package failed. I'll remove it